doc/reference/c2context.rst

```diff
@@ -0,0 +1,9 @@
+Caterva2 context manager
+========================
+
+
+.. currentmodule:: caterva2
+.. autosummary::
+    :toctree: autofiles
+
+    c2context
```

Comment:
I don't know if creating a doc module for just a single function (context in this case) is a good idea.  Wouldn't it be better to move that into their own auth section (together with adduser, deluser and listusers and possibly others)?

Reply:
The context is applied to all (or almost all) the functions in top level as well as the classes Root, File and Dataset...